decidim-proposals/app/forms/decidim/proposals/admin/proposal_form.rb

```diff
@@ -22,6 +22,8 @@ class ProposalForm < Decidim::Form
         attribute :suggested_hashtags, Array[String]
         attribute :photos, Array[String]
         attribute :add_photos, Array
+        attribute :card_image
+        attribute :remove_card_image
```

Comment:
Is this attribute being used? Shouldn't it be in the form as a checkbox? where is the removal of the image finally done?

```suggestion
        attribute :remove_card_image, Boolean
```

Reply:
Yes, it is. And it's a checkbox already, that removes the image.